examples/misc/agent_services.cfg

```diff
@@ -0,0 +1,29 @@
+
```

Comment:
I don't really understand what this config and the example below are for: there are many values defined which are not used?

Reply:
I inteded to read this config file as an input for service resource config. Ranks_per_node is the new config which doesn't exist as of now